components/atomic/buttons/ButtonControl/patterns/ButtonControlDrawer.tsx

```diff
@@ -7,9 +7,9 @@ type DrawerLinkProps = React.ComponentProps<typeof DrawerLink>;
  * A button control that opens a drawer
  */
 const ButtonControlDrawer = forwardRef(
-  ({ drawer, icon, children }: Props, ref) => {
+  ({ drawer, drawerQuery, icon, children }: Props, ref) => {
     return (
-      <DrawerLink drawer={drawer} passHref>
+      <DrawerLink drawer={drawer} drawerQuery={drawerQuery} passHref>
```

Comment:
While I didn't end up using drawerQuery values for this form, I did find a bug where the drawerQuery wasn't getting passed to the drawer helper. This change, [ButtonControlGroup](https://github.com/NGLPteam/wdp-admin-ui/pull/157/files#diff-c1de5c6e46571289a39fa05e934338e8d354b3b220e8f2c4704cc4a47ddb89beR27), and [DrawerLink](https://github.com/NGLPteam/wdp-admin-ui/pull/157/files#diff-14a158117345830464bb9185b4a81f585fb29f9932272d3ec15202aff399d47aR20) changes fix that bug.